hasura/metadata/actions.yaml

```diff
@@ -1,15 +1,29 @@
 actions:
+- name: createUser
+  definition:
+    kind: synchronous
+    handler: '{{HASURA_ACTION_BASE_URL}}/create_user'
+    forward_client_headers: true
+    headers:
+    - name: verification_key
+      value_from_env: HASURA_EVENT_SECRET
+  comment: creates a user in a circle and creates a profile if none exists
```

Comment:
missing permissions for role - user, superadmin. 
to be able to trigger this mutation. 
For this also i wonder if you are able to add an optional condition to the middleware, only users of `role` = 1 of the same `circle_id` are allowed to use this mutation. A couple of other mutations (to be migration) would also require this authentication e.g updating users/ circles